Gemfile

```diff
@@ -17,6 +17,11 @@ gem 'sass-rails', '~> 5.0', '>= 5.0.6'
 gem 'uglifier', '~> 3.2'
 gem 'uswds-rails', '~> 1.3'
 
+source 'https://rails-assets.org' do
+  gem 'rails-assets-jquery', '3.2.1'
+  gem 'rails-assets-owl-carousel2', '2.2.1'
+end
```

Comment:
> is there any licensing issues here?

Shouldn't be an issue, no. Both projects use the MIT License.

> if we're pulling jQuery into the project, then we could (read: should) use it on the FAQ page to slim down the code there.

Yeah, definitely. Once this PR and/or #47 are in, we can refactor the FAQ JS to use jQuery.